### PR TITLE
Added base, header, footer, animation styling, background images.

### PR DESCRIPTION
**Pull request by adammarostica**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,36 +1,59 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Neonderthaw&family=Permanent+Marker&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href=".\styles\style.css" />
+    <title>Bar Cart - Home Page</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="heading-text">
```
Comment (jhshim85) [inline index.html:18]: for BEM use, isn't it block__element--modifier?
eg) <h1 class="heading__text">
      <span class="heading__text--bar>?

Comment (jhshim85) [inline index.html]: for BEM use, isn't it block__element--modifier?
```suggestion
     <h1 class="heading__text">
     <span class="heading__text--bar>?
```

File: styles/sass/partials/_animations.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@keyframes flicker {
```
Comment (jhshim85) [inline styles/sass/partials/_animations.scss:1]: I LOVE how you styled our bar cart!!!

File: styles/sass/partials/_base.scss [hunk 3]
```diff
@@ -1,8 +1,23 @@
 body {
+  position: relative;
   min-height: 100vh;
   display: grid;
   grid-template-rows: auto 1fr auto;
   grid-template-columns: 100%;
+  // Body background applied to a before element so that a 'filter' can be applied to the background, but not the rest of the body.
```
Comment (jhshim85) [inline styles/sass/partials/_base.scss]: Great use of before pseudo element!
May I suggest linear-gradient on the background-image though?
```suggestion
background-image: linear-gradient(90deg, rgba(40, 40, 40) 80%, rgba(35, 35, 35) 80%), url();
```

File: styles/sass/partials/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  color: $white;
+  text-rendering: optimizeLegibility;
+}
+
+h1, h2, h3, p {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
```
Comment (jhshim85) [inline styles/sass/partials/_typography.scss:16]: love the font!

File: styles/style.css [hunk 5]
```diff
@@ -235,12 +226,46 @@ html {
...
+}
+
 body {
+  position: relative;
   min-height: 100vh;
   display: grid;
   grid-template-rows: auto 1fr auto;
   grid-template-columns: 100%;
 }
+body::before {
```
Comment (jhshim85) [inline styles/style.css:256]: I think if we use linear-gradient on background image, this can be voided.

**Review by jhshim85 — COMMENTED** [review]
SUPERB JOB!!!
Please see the comments regarding BEM use and background-image styling.
Thanks!

File: index.html [hunk 6]
```diff
@@ -50,9 +50,8 @@ <h1 class="heading-text">
       </section>
     </main>
     <footer>
-      <p>
-        Made by <span>Jay</span> and <span>Adam</span> at
-        <span><a href="https://junocollege.com/" class="juno-link">Juno College</a></span>
+      <p class="footer-text">
```
Comment (jhshim85) [inline index.html]:
```suggestion
      <p class="footer__text">
      <a href="https://junocollege.com/" class="footer__text--juno">
```

File: styles/sass/partials/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,13 @@
+footer {
+  padding: 10px;
+}
+
+.footer-text {
```
Comment (jhshim85) [inline styles/sass/partials/_footer.scss]:
```suggestion
.footer__text / &--juno
```

File: styles/sass/partials/_variables.scss [hunk 8]
```diff
@@ -1,2 +1,3 @@
 $white: #fff;
-$neonPink: #f09;
\ No newline at end of file
+$neonPink: #f09;
+$junoorange: #ea593e;
```
Comment (jhshim85) [inline styles/sass/partials/_variables.scss:3]: LOVE the colour selection!

**Review by jhshim85 — COMMENTED** [review]
Love the overall changes on footer styling!

File: styles/style.css (hunk 5 above)
Comment (adammarostica) [reply styles/style.css:256]: I'll have to try that.

**Pull request merged**